PowerUp/app/src/main/java/powerup/systers/com/db/AbstractDbAdapter.java

```diff
@@ -165,9 +191,9 @@ public void onCreate(SQLiteDatabase db) {
 
 		@Override
 		public void onUpgrade(SQLiteDatabase db, int arg1, int arg2) {
-			db.execSQL("DROP TABLE IF EXISTS Question");
-			db.execSQL("DROP TABLE IF EXISTS Answer");
-			db.execSQL("DROP TABLE IF EXISTS Scenario");
+			db.execSQL("DROP TABLE IF EXISTS "+PowerUpContract.QuestionEntry.TABLE_NAME);
```

Comment:
Leave a space before and after `+`. if you run Code->Reformat Code on Android Studio, you will get the right formatting